CDCgov/prime-reportstream
# Engagement/angela/12052/ri up

This PR ...

Adds receiver transforms and contains translated settings for RI.

Test Steps:
1. Use the `./prime login --env prod` and `./prime multiple-settings get --env prod --filter ri-phd -l -a` commands to pull the RI settings for Covid pipeline from Production and load them to the DB
2. Use the `./prime multiple-settings set prime-router/metadata/hl7_mapping/receivers/STLTs/RI/ri-phd.yml` to load the settings for the UP
3. Do a `./gradlew clean package` and restart the app
4. Run the files through using postman or curl [Covid](https://github.com/CDCgov/prime-reportstream/files/13667787/RItest.csv) [UP](https://github.com/CDCgov/prime-reportstream/files/13667798/RItest.txt) (There are Postman collections for both the covid pipeline and the UP in the ticket if you need them, but the data with them is incorrect)
5. Compare the resulting files, they should be fairly similar. There are a few differences due to the way Simple Report is sending data, but they should be substantially the same.

## Changes
- Created RI-receiver-transform.yml to handle RI's UP receiver transforms
- Created ri-phd.yml to hold RI's settings for UP (minus the transport settings, which will need to be grabbed from the app when ready to put into Staging and Prod)

## Checklist

### Testing

## Linked Issues
- Fixes #12052
